deadmatterconfig.json

```diff
@@ -0,0 +1,128 @@
+        "Description":"Sets the server name",
+        "Keywords":"server,name,steamservername",
+        "FieldName":"SteamServerName",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"SteamServerName",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"AMP Powered Server",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Lobby Header",
+        "Category":"Dead Matter Server Settings",
+        "Description":"Sets the server title displayed in the lobby",
+        "Keywords":"server,name,lobby,servername",
+        "FieldName":"ServerName",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"/Script/DeadMatter.DMGameSession.ServerName",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"AMP Powered Server",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"MOTD",
+        "Category":"Dead Matter Server Settings",
+        "Description":"Sets the server Message of the Day (MOTD)",
+        "Keywords":"server,motd,welcome,message,day",
+        "FieldName":"MOTD",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"/Script/DeadMatter.DMGameSession.MOTD",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"This server is powered by AMP",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Player Limit",
+        "Category":"Dead Matter Server Settings",
+        "Description":"Sets the maximum number of players allowed on the server",
+        "Keywords":"max,players,limit,maxplayers",
+        "FieldName":"$MaxUsers",
+        "InputType":"number",
+        "MinValue":"1",
+        "IsFlagArgument":false,
+        "ParamFieldName":"MaxPlayers",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"32",
+        "Placeholder":"32",
+        "Suffix":"players",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Enable PvP",
+        "Category":"Dead Matter Server Settings",
+        "Description":"If set, PvP will be enabled",
+        "Keywords":"pve,pvp",
+        "FieldName":"PVP",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"/Script/DeadMatter.DMGameSession.PVP",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"true",
+        "EnumValues":{
+            "False":"false",
+            "True":"true"
+        }
+    },
+    {
+        "DisplayName":"Additional Command Line Arguments",
+        "Category":"Dead Matter Server Settings",
+        "Description":"Sets additional command arguments to start the server with that are not otherwise set by AMP",
+        "Keywords":"switches,command,line,arguments",
+        "FieldName":"ServerSwitches",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"ServerSwitches",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"",
+        "Placeholder":"-hardcore -heartbeat",
```

Comment:
Why not add settings for these known command line args?

Reply:
The format required is different and not -{0}="{1}" like the others. Is it possible to have different methods? e.g. hardcore mode is just `-hardcore`

Reply:
Of course. See the Ark template for examples. `IsFlagArgument` doesn't work properly, so you just include the FieldName (eg `{{hardcore}}` in the command line args directly the kvp

Reply:
Changed in latest commit